api/package/get.js

```diff
@@ -1,12 +1,63 @@
-const { MSGS: { INTERNAL_SERVER_ERROR } } = require('../../helpers/constants')
+const { MSGS: { INTERNAL_SERVER_ERROR }, USER_WEB_SESSION_COOKIE } = require('../../helpers/constants')
 
```

Comment:
before we merge this, can we get package/get coverage up to 100 ?

Reply:
sad.... had a test.only in the test file :(